Backend/DataAccessLayer/Database/dbContext.cs

```diff
@@ -13,13 +13,14 @@ public class DatabaseContext : DbContext
     {
         public DatabaseContext()
         {
-            this.Database.Connection.ConnectionString = "Data Source=localhost;Initial Catalog=NightWatchDB;Integrated Security=True";
+            this.Database.Connection.ConnectionString = "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=NightWatchDB;Integrated Security=True";

```

Comment:
This shouldn't change

Reply:
none of my tests will pass without this change

Reply:
Also, the edited version of that line is what is currently in the master branch, so I'm not sure why this isn't valid.

Reply:
You can keep it on your local machine that way, just don't include it in the PR

Reply:
Ok, I will not include this in the pr.

Reply:
This still seems to be here.